openedx/core/lib/api/view_utils.py

```diff
@@ -148,17 +148,22 @@ def clean_errors(error):
         return {key: clean_errors(value) for key, value in error.items()}
 
 
-def add_serializer_errors(serializer, data, field_errors):
+def add_serializer_errors(serializer, data, field_errors, specify_user_message_for_fields=None):
     """Adds errors from serializer validation to field_errors. data is the original data to deserialize."""
     if not serializer.is_valid():
         errors = serializer.errors
         for key, error in iteritems(errors):
             error = clean_errors(error)
+            if specify_user_message_for_fields and key in specify_user_message_for_fields:
+                user_message = _(' '.join(error))
```

Comment:
It will not get translated like this.

Reply:
You might need to translate this validation error directly to achieve this. https://github.com/edx/edx-platform/blob/4910cfa1808a00fac08f296fb57dbb69a16d51bd/openedx/core/djangoapps/user_api/accounts/serializers.py#L253